src/dynarmic/backend/x64/a32_interface.cpp

```diff
@@ -322,4 +322,8 @@ void Jit::DumpDisassembly() const {
     Common::DumpDisassembledX64(impl->block_of_code.GetCodeBegin(), size);
 }
 
+std::vector<std::string> Jit::Disassemble() {
```

Comment:
```suggestion
std::vector<std::string> Jit::Disassemble() const {
```

Since this shouldn't alter the state of the JIT